library.rb

```diff
@@ -0,0 +1,46 @@
+require 'json'
+
+# Load data from files or create new empty data if files are missing or empty
+books = if File.exist?('books.json') && !File.zero?('books.json')
+          JSON.parse(File.read('books.json'), symbolize_names: true)
+        else
+          []
+        end
+people = if File.exist?('people.json') && !File.zero?('people.json')
+           JSON.parse(File.read('people.json'), symbolize_names: true)
+         else
+           []
+         end
+rentals = if File.exist?('rentals.json') && !File.zero?('rentals.json')
+            JSON.parse(File.read('rentals.json'), symbolize_names: true)
+          else
+            []
+          end
```

Comment:
- [ ] Changes requested by the previous reviewer were not addressed still when I exit the application data are not saved in the `JSON` files please address that to be ready to move forward